heimdall-middleware-spec/src/test/java/br/com/conductor/heimdall/middleware/util/helpermock/call/RequestMockTest.java

```diff
@@ -0,0 +1,59 @@
+package br.com.conductor.heimdall.middleware.util.helpermock.call;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import java.util.HashMap;
+
+public class RequestMockTest {
+
+  @Test
+  public void testHeaderOutputNotNull() {
```

Comment:
change the test case name to testHeader. So the test case name is actually the method you are testing